README.md

```diff
@@ -15,39 +15,72 @@
   <img src="assets/res.webp"/>
 </p>
 
-# About
-
 This GTK theme is based on the [Colloid](https://github.com/vinceliuice/Colloid-gtk-theme) theme made by [Vinceliuice](https://github.com/vinceliuice)
 
-## Usage
+# Usage
 
 ### Requirements
 
-- GTK `>=3.20`
+- GTK >=3.20
+- `python3`
 - `gnome-themes-extra` (or `gnome-themes-standard`)
 
-### Installation
+If you want to build the theme, make sure these packages are installed:
+- `sassc`
+- `inkscape`
+- `optipng`
+
+In order to make sure you're running Python 3, run ``python --version`` and it should output ``Python 3.x.x``
 
-1. Download and extract the theme zip from [releases](https://github.com/catppuccin/gtk/releases/).
-2. Move the theme folder to **".themes"** in your home directory. **(~/.themes)** (Skip this step if you are using the AUR package)
-3. Select the downloaded theme via your desktop specific tweaks application (**gnome-tweaks** on Gnome 3+).
+### Installing the theme manually
 
-### For Arch Linux users
+1. Download and extract the theme zip from the [releases](https://github.com/catppuccin/gtk/releases/) page.
+2. Move the theme folder to the **~/.local/share/themes** directory (Skip this step if you're using the AUR package)
```

Comment:
```suggestion
2. Move the theme folder to the `~/.local/share/themes` directory (skip this step if you're using the AUR package).
```

Reply:
i think that capital is fine though

Reply:
Text in parenthesis should be lowercase if it is in the middle/end of a sentence.